app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -270,25 +271,26 @@
     private PlayQueueAdapter playQueueAdapter;
     private StreamSegmentAdapter segmentAdapter;
 
-    @Nullable private MediaSourceManager playQueueManager;
+    @Nullable private PlayQueueReactor playQueueManager;
 
     @Nullable private PlayQueueItem currentItem;
-    @Nullable private MediaSourceTag currentMetadata;
+    @Nullable private MediaItemTag currentMetadata;
     @Nullable private Bitmap currentThumbnail;
 
     /*//////////////////////////////////////////////////////////////////////////
     // Player
     //////////////////////////////////////////////////////////////////////////*/
 
-    private SimpleExoPlayer simpleExoPlayer;
+    private ExoPlayer simpleExoPlayer;
     private AudioReactor audioReactor;
     private MediaSessionManager mediaSessionManager;
     @Nullable private SurfaceHolderCallback surfaceHolderCallback;
 
-    @NonNull private final CustomTrackSelector trackSelector;
```

Comment:
`CustomTrackSelector` interface has conflicts now, `DefaultTrackSelector` already fixed the issue with language selection, so making the switch here.